src/app/pcpolicy/pcpolicy.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+import { DashboardService } from '../services/dashboard.service';
+
+@Component({
+  selector: 'app-pcpolicy',
+  templateUrl: './pcpolicy.component.html',
+  styleUrls: ['./pcpolicy.component.scss']
+})
+
+export class PcpolicyComponent implements OnInit {
+  email: String;
+  public position= 'col-md-10 col-md-offset-2';
+  constructor(private _dashboardService: DashboardService) { }
+
+  ngOnInit() {
+    this._dashboardService.getUserInfo().subscribe(response => {
+        this.email = response.user.email;
+    });
+  }
+
+  toggle() {
+    this.position === 'col-md-10 col-md-offset-2' ? this.position = 'col-md-12 pc_policy_body' :
```

Comment:
you can actually factor out `this.position` to something below:
```
this.position = (this.position === 'col-md-10 col-md-offset-2') ? 'col-md-12 pc_policy_body' : 'col-md-10 col-md-offset-2';
```
the bracket isn't actually necessary, just eye-candy to make separations of the statement